commit 8bb1b84cb87ea5cc24a43a4ce3279e212daf8a75
Author:     Yuhong Wen <yuhongw@nvidia.com>
AuthorDate: Tue May 28 17:43:14 2024 -0400
Commit:     GitHub <noreply@github.com>
CommitDate: Tue May 28 14:43:14 2024 -0700

    Missing sj heartbeat (#2583)
    
    * Added the missing SJ heartbeat, fixed the early abort_job command issue.
    
    * codestyle fix.
    
    * Removed the no use constant.
    
    * Refactored.
    
    ---------
    
    Co-authored-by: Yuan-Ting Hsieh (謝沅廷) <yuantingh@nvidia.com>
---
 nvflare/private/fed/server/server_engine.py | 2 --
 1 file changed, 2 deletions(-)

diff --git a/nvflare/private/fed/server/server_engine.py b/nvflare/private/fed/server/server_engine.py
index 60f28bb7..77224cd9 100644
--- a/nvflare/private/fed/server/server_engine.py
+++ b/nvflare/private/fed/server/server_engine.py
@@ -317,8 +317,6 @@ class ServerEngine(ServerEngineInternalSpec):
         return ""
 
     def abort_app_on_server(self, job_id: str, turn_to_cold: bool = False) -> str:
-        if job_id not in self.run_processes.keys():
-            return "Server app has not started."
 
         self.logger.info("Abort the server app run.")
         command_data = Shareable()
